src/python/grpcio/grpc/__init__.py

```diff
@@ -299,6 +299,23 @@ def add_callback(self, callback):
     raise NotImplementedError()
 
 
+############################  gRPC Metadata  ##################################
+
+
+class Metadatum(six.with_metaclass(abc.ABCMeta, tuple)):
+  """A single entry in a Metadata collection.
+  
+  A metadata entry is a 2-tuple where the first entry is the key and the
```

Comment:
Is there any reason for a Metadatum to be exposed to applications as a sequence? Do we expect applications to iterate over its two values, the way they would iterate over an arbitrary-length sequence?